2016-09/september-07.md

```diff
@@ -0,0 +1,80 @@
+## August 31 ([discuss](https://github.com/webpack/meeting-notes/pull/13))
+
+### Attendees
+
+* [Tobias Koppers](http://github.com/sokra)
+* [Sean Larkin](http://github.com/thelarkinn)
+* [Johannes Ewald](http://github.com/jhnns)
+* [Juho Vepsäläinen](http://github.com/bebraw)
+
+
+### Agenda
+
+#### Documentation Updates
+* Juho: Migrated a significant grouping of issues labeled `question` or `documentation` from [webpack/webpack](github.com/webpack/webpack) over to the [new docs page](github.com/webpack/webpack.js.org).
+
+* PR's are now in for adding helpful documentation linters and plugins
+	* proselint: will help with grammar and has settings for rules etc. [#96](https://github.com/webpack/webpack.js.org/pull/96)
+
+	* ESLINT module for codeblocks for markdown documents. This will ensure that our code examples are consistent across documents etc. [MERGED #97](https://github.com/webpack/webpack.js.org/pull/97)
+
+	* Juho also suggested we look into `alex` which works with proselint on spelling and linting specific words based on topic (offensive, abbreviations, etc.)
+
+	* Don't want linting too strict, just enough to provide, autoformatting, and eliminate the number of round of reviews for PR's.
+
+* The topic of Cont. Deployment came up for our documentation. We viewed some other pages and it apeard Codeship is a popular option as well as Travis but requires more work. An issue will be created in the docs repo. 
+
+#### Publication Updates
+* Upon feedback, we discovered that Housing.com's engineering team had published an excellent article on medium about their success's with webpack. Sean suggested that we open up the option for authors who are interested, to post success stories and awesome information under our publication (after review). The team decided this would be fine since we have been slacking on our medium publication as of late. 
+
+#### Domain
+* We discussed between `webpack.js.org` and `webpack.github.io`. After we did the research we decided that we would setup `webpack.js.org`. Tobias is going to tackle this a bit later today. 
+
+#### Analytics 
+* Sean asked whether or not we have page analyitcs on our docs site currently. Tobias gave access to all the analytic data. Sean also brought up the idea of having webpack the tool ask if users would let us submit analyitcs data. Tabeled discussion as its a severe nice-to-have.
```

Comment:
Just warning you: Beware of the shit storm, make it very obvious that it's opt-outable. (see this very long discussion: https://github.com/atom/atom/issues/4966)